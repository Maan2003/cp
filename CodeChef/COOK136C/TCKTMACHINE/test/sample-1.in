2
7 10
2 10 5
3 10 5
4 9 1
5 9 5
6 8 1
7 8 1
8 8 1
3 10
2 10 9
2 9 1
8 9 1
