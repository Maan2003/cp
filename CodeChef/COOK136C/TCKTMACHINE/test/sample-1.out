1 3 4 4 7 7 7
1 3 3
